src/organisms/subject-attend.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
```
`React` の import は不要です.